node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoreCollector.java

```diff
@@ -49,6 +52,17 @@ Path readBinPathFallback(NodeAgentContext context, Path coredumpPath) {
         return Paths.get(matcher.group("path").split(" ")[0]);
     }
 
+    Path GDBPath(NodeAgentContext context) {
+        DockerImage image = context.node().currentDockerImage().get();
```

Comment:
`currentDockerImage` is not guaranteed to be set (f.ex. if core is dumped while starting or stopping the container), I suggest defaulting to the RHEL 7 variant.